menpo/transform/homogeneous/rotation.py

```diff
@@ -151,6 +151,84 @@ def init_3d_from_quaternion(cls, q):
         r = cls.init_identity(n_dims=3)
         return r.from_vector(q)
 
+    @classmethod
+    def init_from_3d_ccw_angle_around_x(cls, theta, degrees=True):
+        r"""
+        Convenience constructor for 3D CCW rotations around the x axis
+
+        Parameters
+        ----------
+        theta : `float`
+            The angle of rotation about the origin
+        degrees : `bool`, optional
+            If ``True`` theta is interpreted as a degree. If ``False``, theta is
+            interpreted as radians.
+
+        Returns
+        -------
+        rotation : :map:`Rotation`
+            A 3D rotation transform.
+        """
+        if degrees:
+            # convert to radians
+            theta = theta * np.pi / 180.0
+        return Rotation(np.array([[ 1,             0,              0],
+                                  [ 0, np.cos(theta), -np.sin(theta)],
+                                  [ 0, np.sin(theta), np.cos(theta)]]),
+                        skip_checks=True)
+
+    @classmethod
+    def init_from_3d_ccw_angle_around_y(cls, theta, degrees=True):
+        r"""
+        Convenience constructor for 3D CCW rotations around the y axis
+
+        Parameters
+        ----------
+        theta : `float`
+            The angle of rotation about the origin
+        degrees : `bool`, optional
+            If ``True`` theta is interpreted as a degree. If ``False``, theta is
+            interpreted as radians.
+
+        Returns
+        -------
+        rotation : :map:`Rotation`
+            A 3D rotation transform.
+        """
+        if degrees:
+            # convert to radians
+            theta = theta * np.pi / 180.0
+        return Rotation(np.array([[ np.cos(theta), 0, np.sin(theta)],
+                                  [             0, 1,             0],
+                                  [-np.sin(theta), 0, np.cos(theta)]]),
+                        skip_checks=True)
+
+    @classmethod
+    def init_from_3d_ccw_angle_around_z(cls, theta, degrees=True):
+        r"""
+        Convenience constructor for 3D CCW rotations around the z axis
```

Comment:
Not sure what you mean? `ccw` infers Counter ClockWise, so to me it seems fully specified. I guess we could state that XYZ is a RH coordinate system but it feels that this assumption is made in far more places than just this in menpo...

P.S. think it's a little dangerous to state what 'up' is - I guess it doesn't really have a meaning here. If I have a facial mesh where `+z` is up, then to spin the head around I would use to `init_from_3d_ccw_angle_around_z` method. If the mesh is orientated with `+y` as up, I'd use the `init_from_3d_ccw_angle_around_y` to spin that one in a CCW fashion...unless I'm missing your meaning here?